_pages/en/saas-and-cloud-strategy.md

```diff
@@ -0,0 +1,230 @@
+---
+layout: default
+title: SaaS vs Cloud Strategy
+lang: en
+ref: documents
+status: posted
+permalink: /saas-vs-cloud.html
+---
+
+## Context
+
+The process of creating this strategy begun due to the team receiving a request to help answer the question as to whether one should procure a "Cloud-based SaaS Solution" or a "DevOps In-house Build Cloud deployed solution." This document is attempting to assist in clarifying the terms used in the question, and help guide this decision, and similar decisions that will need to be made going forward, by defining the strategy which will clarify how and why each technology should be used to assist the organization in better delivering on its mandate.
+
+## The Question of Cloud
+
+### Goals
+
+In order to align with the [Digital Operations Strategic Plan: 2018-2022](https://www.canada.ca/en/government/system/digital-government/digital-operations-strategic-plan-2018-2022.html), which includes the following as a "guide [as to] how the government and public service work on a day-to-day basis"
+
+> Iterate and improve frequently: Develop services using agile, iterative and user-centred methods. Continuously improve in response to user needs. Try new things, start small and scale up.
+
+Further, In the [Government of Canada Cloud Adoption Strategy: 2018 update](https://www.canada.ca/en/government/system/digital-government/modern-emerging-technologies/cloud-services/government-canada-cloud-adoption-strategy.html) several benefits of cloud are listed, including *service performance*, *security*, *innovation*, *agility*, and *elasticity*.
+
+In order to become a more agile organization ESDC must be able to respond quickly to the needs of citizens or operational issues. 
+
+Industry best practices, such as those exposed in the [2019 DORA State of DevOps Report](https://www.google.ca/url?sa=t&rct=j&q=&esrc=s&source=web&cd=2&cad=rja&uact=8&ved=2ahUKEwiN7Yfn5MTlAhWhnOAKHTdYDmkQFjABegQIAhAC&url=https%3A%2F%2Fcloudplatformonline.com%2Frs%2F248-TPC-286%2Fimages%2FDORA-State%2520of%2520DevOps.pdf&usg=AOvVaw2PLSLELi2td1WLLLoiqzcl), promote frequent deployments and low lead times.
```

Comment:
Don't link to a google cache, link to the current StateOfDevops
https://cloud.google.com/devops/state-of-devops/

If you quote a particule report, Say which version of that report is (e.g. 2018 report). But quoting an outdated report may play against you in the credibility of the statement